ProjectStructureAndRoles.md

```diff
@@ -75,6 +75,10 @@ CAMARA commonalities shall be processed in a Working Group “Commonalities”.
 
 All Sub Projects must comply with the work in the commonalities Working Group.
 
+**Identity and Consent Management**
+
+In the Identity and Consent Management Working Group solutions are defined and developed to identify users/subscribers, to capture, store and manage user consent, and to make sure that end user experience of using the API is not compromised.
+
```

Comment:
```suggestion
**Identity and Consent Management**

The Identity and Consent Management Working Group, operating under the supervision of the Technical Steering Committee (TSC), focuses on defining and developing comprehensive solutions for user/subscriber identification, consent capture, storage, and management. These solutions ensure that user experience with the API remains seamless and uncompromised.

All Sub Projects must comply with the work in the "Identity and Consent Management" Working Group. The Working Group's efforts include defining API access parameters and developing the Security and Interoperability Profile, which encompasses CAMARA definitions for authentication and authorization (AuthN/AuthZ) flows.
```
Does this look good? Please feel free to change anything you like.

Reply:
Fair point to express the same level of governance and compliance for Identity and Consent Management.

Small changes in my proposal below:
* I haven't understood what is meant with "defining API access parameters" so I left it out. 
* Proposal to not start a new explanation after the sentence "All Sub Projects must comply with the work in the Identity and Consent Management Working Group.
* replaced "uncompromised" (which my syntax check does not like) with "protected".

```suggestion
**Identity and Consent Management**
The Identity and Consent Management Working Group, operating under the supervision of the Technical Steering Committee (TSC), focuses on defining and developing comprehensive solutions for user/subscriber identification, consent capture, storage, and management. These solutions ensure that user experience with the API remains seamless and protected. 

The Working Group's efforts include developing the Security and Interoperability Profile, which encompasses CAMARA definitions for authentication and authorization (AuthN/AuthZ) flows.

All Sub Projects must comply with the work in the Identity and Consent Management Working Group.
```

Reply:
I do not want it to be too complicated but usually there is no "user identification".
Telcos do identify the subscriber but we do not identify the user. At least not what commonly is understood as the "user".

Also, I would rephrase making it clearer that "storage, and management" are related to consent.

> The Identity and Consent Management Working Group, operating under the supervision of the Technical Steering Committee (TSC), focuses on defining and developing privacy-first solutions for subscriber identification, and user authentication, and the purpose of the API access, and user consent capture, storage, and management. These solutions ensure that the API access is always authorized and secure. 

That got somewhat lengthy...